models/src/main/protobuf/CasperMessage.proto

```diff
@@ -185,7 +185,8 @@ message BlockMessage {
   bytes                  sig            = 7; // signature generated by signing `hash(hash(justification) concat blockHash)`.
   string                 sigAlgorithm   = 8; // name of the algorithm used to sign
   string                 shardId        = 9; // identifier of the shard where the block was created
-  bytes extraBytes = 10;
+  int32                  creatorDist    = 10; // distance to the creator of the block
```

Comment:
I think it will be confusing if we start placing things that aren't included in the blockHash in BlockMessage. We should probably move creatorDist and the sig out into a different message.